src/main/kotlin/apply/application/ApplicationFormService.kt

```diff
@@ -50,9 +50,11 @@ class ApplicationFormService(
         )
         applicationForm.update(request.referenceUrl, answers)
         if (request.isSubmitted) {
+            require(!applicationFormRepository.existsByApplicantIdAndSubmittedTrue(applicantId)) {
```

Comment:
```suggestion
require(applicationFormRepository.existsByApplicantIdAndSubmittedFalse(applicantId)) {
```
개인적으로 ! 연산 보다는 바로 사용하는게 읽기 좋다고 생각하는데 취향이니 참고만해주세요 :)

Reply:
제안 해주신 쿼리 메소드는 제출하지 않는 지원서가 하나라도 존재하면 `true`이기 때문에 모든 지원서를 제출할 수 있어요.

Reply:
그렇네요...! 제가 말한대로 기능하려면 `notExistsByApplicantIdAndSubmittedTrue`가 필요하겠네요 😅
지금 그대로 사용해야겠네요 👍